src/components/plugins/Categories.tsx

```diff
@@ -10,6 +10,9 @@ const createRows = (categories: Category[]) => {
     return [];
   }
   const sorted = categories.sort((left, right) => {
+    if(left.displayName === "All Plugins") return -1;
+    if(right.displayName === "All Plugins") return 1;
```

Comment:
Please use proper formatting, do not use the if syntax without braces and use the identifier for matching not the display name.

```suggestion
    if (left.name === "all"){
      return -1;
    }
    if (right.name === "all"){
      return 1;
    }
```